updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+== Updating a cluster using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster by using the web console. You can the update a cluster between minor versions. For more information, see the following items:
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-within-minor.adoc#update-changing-update-server-web_updating-cluster-within-minor[Changing the update server by using the web console]
+
+[id="updating-clusters-overview-update-cluster-using-cli"]
+== Updating a cluster within a minor version using the command-line interface (CLI)
+xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version using the CLI]: You can update an {product-title} cluster by using the `oc`. You can the update a cluster between minor versions. For more information, see the following actions:
+
+* xref:../updating/updating-cluster-cli.adoc#update-upgrading-cli_updating-cluster-cli[Updating a cluster by using the CLI]
+* xref:../updating/updating-cluster-cli.adoc#update-changing-update-server-cli_updating-cluster-cli[Changing the update server by using the CLI]
+
+[id="updating-clusters-overview-update-cluster-with-rhel-compute-machines"]
+== Updating a cluster that includes {op-system-base} compute machines
+xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes {op-system-base} compute machines]:  You can update an {product-title} cluster. If your cluster contains {op-system-base-full} machines, you must update those machines. You can perform the following actions:
+
+* xref:../updating/updating-cluster-rhel-compute.adoc#update-upgrading-web_updating-cluster-rhel-compute[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute-hooks[Optional: Adding hooks to perform Ansible tasks on {op-system-base} machines]
+* xref:../updating/updating-cluster-rhel-compute.adoc#rhel-compute-updating-minor_updating-cluster-rhel-compute[Updating {op-system-base} compute machines in your cluster]
+
+[id="updating-clusters-overview-update-restricted-network-cluster"]
+== Updating a restricted network cluster
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network {product-title} cluster by using the `oc`. You can also update a restricted network {product-title} cluster by using the OpenShift Update Service. For more information, see the following items:
```

Comment:
We release Update service images for deploying it in disconnected environments. Here is the documentation for your reference https://docs.openshift.com/container-platform/4.10/updating/updating-restricted-network-cluster.html#update-restricted-network-cluster-update-service

Reply:
@LalatenduMohanty So can we write `You can also update a restricted network {product-title} cluster by using the OpenShift Update Service`? sorry I am confused

Reply:
If customers have installed running OSUS service for the disconnected environment then  they can update clusters in that disconnected environment similar fashion to connected cluster using OSUS. Or If customers want to use OSUS in disconnected environment then they need to run OSUS in that environment, then point the clusters to the OSUS instance and then then they can update clusters in that disconnected environment similar fashion to connected cluster using OSUS.

Reply:
> You can also update a restricted network {product-title} cluster by using the OpenShift Update Service

Yes, but provide more context that this is not the default experience but customers can install OSUS and then can use it in the disconnected environment.

Reply:
Addressed

Reply:
Thank you for your updates. Is it still also worth mentioning the mirror repository in this summary paragraph, given that it features so much in the linked sections? It might be useful for the reader to have some context about that specifically.

Reply:
I added the following lines to indicate that they can use mirroring as well. 

```If your registry host cannot access both the internet and the cluster, you can mirror the images to a file system that disconnected from that environment and then bring that host or removable media across that gap. If the local container registry and the cluster are connected to the mirror registry’s host, you can directly push the release images to the local registry.```

Reply:
Thank you for adding that.

Reply:
@LalatenduMohanty can you please review this updated paragraph? Thanks!

Reply:
“registry host” is replaced by “mirror host”.